frontend/src/metabase/visualizations/components/ObjectDetail/ObjectDetail.tsx

```diff
@@ -460,6 +460,7 @@ export function ObjectDetailHeader({
                   icon="chevronup"
                 />
                 <Button
+                  tabIndex={0}
```

Comment:
The `FocusTrap` components seems to work automatically in the other cases I describe in the PR description. However, for the `ObjectDetail` modal, the `FocusTrap` can't seem to find elements to focus on automatically, so we'll automatically focus on this button which will help `FocusTrap` keep the focus in the `ObjectDetail` modal